docs/reference/transactions.md

```diff
@@ -97,7 +98,6 @@ This is a special form of an Asset Transfer Transaction.
 |Field|Required|Type|codec| Description|
 |---|---|---|---|---|
 |<a name="xferasset">XferAsset</a>| _required_ |uint64|`"xaid"`|The unique ID of the asset to be transferred.|
-|<a name="assetsender">AssetSender</a>|_required_|Address|`"asnd"`| The account which is allocating the asset to their account's Asset map.|
```

Comment:
For the same reasons above, I don't know if this field needs to be specifically specified.

Reply:
https://github.com/algorand/docs/pull/452#discussion_r662309668